Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -37,7 +37,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Specify proxy value if it's should be configured on the cluster\n",
+    "# Specify proxy value if applicable\n",
     "%env https_proxy=http://<proxy_host>:<port>\n",
```

Comment:
nit: do we need to comment next line with setting proxy? I order to be consistent with other changes in this PR. But actually in current state it is also fine for me

```suggestion
    "# Uncomment & specify proxy value if applicable\n",
    "# %env https_proxy=http://<proxy_host>:<port>\n",
```

Reply:
Yes I think to be consistent with other changes, we can comment this as well. 

I will update the PR.